base/linalg/triangular.jl

```diff
@@ -1900,14 +1907,10 @@ end
 function sqrtm{T}(A::UnitUpperTriangular{T})
```

Comment:
I think most of the methods in `Base.LinAlg` upwrap the type before indexing. Haven't checked, though.